libraries/Microsoft.Bot.Schema/ActivityEx.cs

```diff
@@ -411,7 +411,7 @@ public Mention[] GetMentions()
         /// <returns>The strongly-typed object; or the type's default value, if the <see cref="ChannelData"/> is null.</returns>
         /// <seealso cref="ChannelData"/>
         /// <seealso cref="TryGetChannelData{TypeT}(out TypeT)"/>
-        public T GetChannelData<T>()
+        public TypeT GetChannelDataObject<T>()
```

Comment:
In this particular case and since we are dealing with a `dynamic` type, I'm more in favor of keeping the exclusion and not changing the method name.